src/background/index.ts

```diff
@@ -224,6 +224,41 @@ function getSelectedText(tabId, cb) {
 // for avoid CORB call, use background and communicate with content script
 // https://stackoverflow.com/questions/54786635/how-to-avoid-cross-origin-read-blockingcorb-in-a-chrome-web-extension
 const messageListener = (request) => {
+
+  if (request.message === "auth_user") {
+    agent.Auth.login(request.user).then(response => {
```

Comment:
stole most of this from authListener, but setting User based on the form submission (request.user). Does this seem right?

Reply:
Yea this looks great!